web-console/knikubevirt/clone-vm/clone-vm.md

```diff
@@ -0,0 +1,43 @@
+# Clone VM
+
+![Offline VM in List View](img/2-0-vms.png)
+
+Cloning a VM allows the user to quickly create an identical copy of a virtual machine while powered off.
+
+![VM List View Clone option in kebab menu](img/2-1-vm-list.png)
+
+Cloning is accessed from the VM List View. All item filters should be turned on by default, ensuring that all VMs are shown to the user.
```

Comment:
should this read "all item filters should be turned off by default"?

Reply:
Right, fixing it in the next commit